qt/python/mantidqt/widgets/test/test_functionbrowser.py

```diff
@@ -0,0 +1,302 @@
+from __future__ import (absolute_import, division, print_function)
+from mock import MagicMock, call
+import unittest
+
+from qtpy.QtWidgets import QApplication
+from qtpy.QtCore import Qt, QPoint, QObject
+from qtpy.QtTest import QTest
+
+from mantid import FrameworkManager, FunctionFactory
+from mantid.fitfunctions import FunctionWrapper
+from mantidqt.utils.qt.testing.gui_window_test import (GuiWindowTest, not_on_windows, get_child, click_on)
+from mantidqt.widgets.functionbrowser import FunctionBrowser
+
+
+skip = unittest.skipIf(not_on_windows(), "It works on windows. I cannot spend too much time trying to "
```

Comment:
Can we create an issue to get this working on other platforms? It doesn't make it easy for other developers to diagnose or see problems if they can't run tests locally.

Reply:
#25695